backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -354,6 +355,38 @@ func (ctx *AppContext) UpdateUserDataFromMySQL() error {
 	return nil
 }
 
+// returns a map of master pubkeys to their account
+func (ctx *AppContext) MyMasterPubKeys() (map[string]string, error) {
+	acctmap, err := ctx.WalletConnection.ListAccounts()
+	if err != nil {
+		log.Errorf("MyMasterPubKeys: ListAccounts rpc failed: %v", err)
+		return nil, err
+	}
+	keymap := make(map[string]string)
+	for acct := range acctmap {
+		// skip imported
+		if acct == "imported" {
+			continue
+		}
+		// There is no construct for getmasterpubkey in dcrd yet so use raw json
```

Comment:
This is dcrwallet, not dcrd

Reply:
the methods are in the dcrd repo, but ill change that